app/ui/setup/src/main/java/com/fsck/k9/ui/setup/EnterEmailAddressFragment.kt

```diff
@@ -0,0 +1,13 @@
+package com.fsck.k9.ui.setup
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class EnterEmailAddressFragment : Fragment() {
```

Comment:
This is not a great name. Suggestions are very welcome.

Reply:
EmailAddressPrompt ?